page.html

```diff
@@ -69,331 +75,320 @@ <h1>Lorem Ipsum is simply dummy text of the printing</h1>
-                    <div class="related_articles_item">
-                        <h3>
+                        </h4>
+                    </li>
+                    <li class="related-articles-item">
+                        <h4>
                             standard dummy text ever since
-                        </h3>
-                    </div>
-                    <div class="related_articles_item">
-                        <h3>
+                        </h4>
+                    </li>
+                    <li class="related-articles-item">
+                        <h4>
                             standard dummy text ever since
-                        </h3>
-                    </div>
-                    <div class="related_articles_item">
-                        <h3>
+                        </h4>
+                    </li>
+                    <li class="related-articles-item">
+                        <h4>
                             Lorem Ipsum has been the industry's
-                        </h3>
-                    </div>
-                </div>
-            </div>
-            <div class="write_comment">
-                <h2>
+                        </h4>
+                    </li>
+                </ul>
+            </section>
+            <section class="write-comment">
+                <h3>
                     write comment
-                </h2>
-                <div class="write_comment_items_container">
-                    <div class="write_comment_item">
-                        <p>
-                            Name :
-                        </p>
-                        <input class="comment_name_input">
-                    </div>
-                    <div class="write_comment_item">
-                        <p>
-                            email :
-                        </p>
-                        <input class="comment_email_input">
-                    </div>
-                    <div class="write_comment_item">
-                        <p>
-                            Website :
-                        </p>
-                        <input class="comment_website_input">
-                    </div>
-                    <div class="write_comment_item">
-                        <p>
-                            Comment :
-                        </p>
-                        <textarea rows="5" class="comment_comment_input"></textarea>
-                    </div>
-                </div>
-                <button>Post comment</button>
-            </div>
-            <div class="comments">
-                <h2>
+                </h3>
+                <form action="/awesome-backend-script.php" name="commentForm" method="post">
+                    <ul class="write-comment-items-container">
+                        <li class="write-comment-item">
+                            <label>Name :</label>
+                            <input id="nameInfo" name="nameInfo" type="text" class="comment-name-input">
```

Comment:
Please add attribute `for` with `nameInfo` value.
Here and further:
The `for` attribute of the `<label>` tag should be equal to the `id` attribute of the related element to bind them together.